alarm_controller.py

```diff
@@ -92,32 +93,51 @@ def run_circ_trans_alarm(self):
                     'No circ trans records found for all of {}'.format(
                         self.yesterday))
 
-    def run_holds_alarm(self):
-        if not self.run_added_tests:
-            return
+    def run_holds_alarms(self):
+        self.logger.info('\nHOLDS\n')
+        self.redshift_client.connect()
+        if self.run_added_tests:
```

Comment:
I'm still a bit unclear around the logic of `run_added_tests` and when/where those are invoked.

Reply:
`run_added_tests` is used for tests that should only be run in prod. For instance, we don't want to sound an alarm when no holds are changed for a day in QA, because this frequently happens (the database isn't updated that frequently). I thought about using an `is_production` boolean or even checking the Redshift database, which is what a previous version of these alarms did, but they're also used in `test_alarm_controller.py` and it felt wrong to set `is_production` to true in a test file.